ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/datasource/interactions/neo4j/bolt/Neo4jBoltDataSourceDialog.java

```diff
@@ -40,8 +44,12 @@ public Neo4jBoltDataSourceDialog(Project project, DataSourcesView dataSourcesVie
 
     public Neo4jBoltDataSourceDialog(Project project, DataSourcesView dataSourcesView) {
         super(project, dataSourcesView);
+        loadingIcon = new AsyncProcessIcon("validateConnectionIcon");
```

Comment:
Can the icon component be added to the form file in GUI Designer (just like the loadingPanel)?

Reply:
done